release-notes/opensearch-security.release-notes-2.11.0.0.md

```diff
@@ -0,0 +1,40 @@
+## 2023-10-18 Version 2.11.0.0
+
+Compatible with OpenSearch 2.11.0
+
+### Enhancements
+* Authorization in Rest Layer ([#2753](https://github.com/opensearch-project/security/pull/2753))
+* Demo configuration script requires admin password ([#3329](https://github.com/opensearch-project/security/pull/3329))
```

Comment:
I've put the title as is, I'm 50-50 about changing it, what do you propose?

Reply:
I think it is fine to leave it like this. Since we do require the admin passwords, but it is not a hard requirement for now. User can still use admin:admin if there is an absent of the `initialAdminPassword`.